test/plugins/wasm_bpf/assets/bootstrap.bpf.c

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
+/* Copyright (c) 2020 Facebook */
```

Comment:
1. Is this file copied from the repo of Facebook?
2. Will this file (or any other examples) in this folder build by the CMake building system? If not, we don't need to copy them into this folder. Please keep a note of how to build the final output in the README instead.

Reply:
1. This file was copied from libbpf's repo, see [here](https://github.com/libbpf/libbpf-bootstrap/blob/master/examples/c/bootstrap.bpf.c)
2. Files under the `assets` folder won't be built by the cmake building system. Where should we put them if they were moved out?

Reply:
Since these files are not used, copying them into the assets folder is unnecessary. Instead, you can directly mention the version/commit you use in the origin repo in the README.

Reply:
> Since these files are not used, copying them into the assets folder is unnecessary. Instead, you can directly mention the version/commit you use in the origin repo in the README.

The build results of there file (the "object files/*.bpf.o files" mentioned in the previous comments) will be used during testing. Should I use cmake or make (use make is more convenient, I think) to build them?

Reply:
Then, trigger it in the cmakefiles (I am okay with using cmake or make). Otherwise, how would you like to get the test files?
Instead of coping assets from the repo, you can use the building system to clone/download and build it.

Reply:
- `runqlat.bpf.c` and `bootstrap.bpf.c` were copied from other repos. I'll write a script to download them when building.
- `simple_map.bpf.c` and `simple_ringbuf.bpf.c` were written by me, I think it's better to keep them in the source tree.

Is this ok?

Reply:
I remove the object files from the source tree, currently the implementation is:
- `simple_map.bpf.c` and `simple_ringbuf.bpf.c` are preserved in the tree. They will be compiled when running the tests through cmake
- CMake will clone the repo https://github.com/eunomia-bpf/wasm-bpf and build the object files of `runqlat.bpf.c` and `bootstrap.bpf.c`. The the object files will be copied into the assets directory for testing purpose.